src/styles/styles.js

```diff
@@ -1255,6 +1267,20 @@ const styles = {
         fontSize: variables.iouAmountTextSize,
     }, 0),
 
+    iouPreviewBox: {
+        borderColor: themeColors.border,
+        borderWidth: 1,
+        borderRadius: variables.componentBorderRadiusCard,
+        padding: 20,
+        marginTop: 16,
+        maxWidth: 400,
+        width: '100%',
+    },
+
+    iouPreviewBoxAvatar: {
+        marginRight: -10,
```

Comment:
Is there a way to implement this component without needing negative margin? Ideally the view that wraps the two avatars will float all the way to the right by default.

Reply:
`MultipleAvatars` has `marginRight: 16` and uses absolute to position avatars. We have to rewrite MultipleAvatars component to achieve that result without using negative margin here.

Do you recommend modifying `MultipleAvatars` to accept style prop where we can set `marginRight: 0` and `width: 46` instead of `marginRight: 16` and `width: 40`?

Reply:
Ah that's fair, I didn't realize we were re-using an existing component. I think what you have is fine then.